src/composables/audit.js

```diff
@@ -24,13 +24,10 @@ const actionPath = (action) => {
   return `audit.action.${category}.${subactionKey}`;
 };
 
-export default () => {
-  const { i18n } = inject('container');
-  return {
-    // Returns a message describing an audit action.
-    actionMessage: (action) => {
-      const path = actionPath(action);
-      return i18n.te(path, i18n.fallbackLocale) ? i18n.t(path) : null;
-    }
-  };
-};
+export default memoizeForContainer(({ i18n }) => ({
+  // Returns a message describing an audit action.
+  actionMessage: (action) => {
+    const path = actionPath(action);
+    return i18n.te(path, i18n.fallbackLocale) ? i18n.t(path) : null;
+  }
+}));
```

Comment:
Today, each `AuditRow` component that uses the `useAudit()` composable ends up creating its own `actionMessage()` function. After this change, the same function will be returned to each component.